docs/developer_setup.md

```diff
@@ -41,3 +41,17 @@ http://localhost:3000 order and serve up the REST API.
   the service UI, which will initiate the UI listening on _http://localhost:3001_, and talking to the REST API at
   _http://[::1]:3000_.  This command will also open a browser page to  _http://localhost:3001/login_.
   - `yarn start`
+
+### Testing
```

Comment:
There is already a section called testing in `contributing.md`, this section might make more sense there